samples/nrf9160/udp/README.rst

```diff
@@ -107,6 +107,11 @@ The following configurations are recommended for low power behavior:
 PSM and eDRX timers are set with binary strings that signify a time duration in seconds.
 See `Power saving mode setting section in AT commands reference document`_ for a conversion chart of these timer values.
 
+.. note::
+   In applications where downlink messaging from cloud to device is expected we recommend to
+   set :kconfig:option:`CONFIG_LTE_PSM_REQ_RAT` a higher value than 0 to ensure data is received before
+   the device enters PSM.
```

Comment:
```suggestion
   In applications where downlink messaging from the cloud to the device is expected, we recommend setting the :kconfig:option:`CONFIG_LTE_PSM_REQ_RAT` option to a higher value than ``0`` to ensure data is received before the device enters PSM.
```

Also, we could move the note in line:115 after line no:106, where we talk about recommendations.